schema/vulntology-json-schema-1.0-draft.json

```diff
@@ -298,6 +298,7 @@
                 "id",
                 "hasImpactMethod",
                 "affectsContext",
+                "hasEntityRole",
```

Comment:
To properly support automation of CVSS values (one of our core goals) we will need to enforce population of this property.